libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKMapper.kt

```diff
@@ -94,6 +94,11 @@ fun asCheckMessageRequest(reqData: ReadableMap): CheckMessageRequest? {
     return null
 }
 
+fun asReverseSwapFeesRequest(reqData: ReadableMap): ReverseSwapFeesRequest? {
+    val sendAmountSats = reqData.getDouble("sendAmountSats")
```

Comment:
The sendAmountSats is defined as optional in the udl. I don't know how the code handle now empty value.
Does it throw error? Does it use default value? In both cases it is not the correct behavior.
Maybe @dangeross can help here?

Reply:
Commented below on this

Reply:
Done in b353354db83fbba408645f2007ca2172cbfeba2c